README.md

```diff
@@ -449,6 +448,8 @@ To learn more about the st-odm release, try:
 
 **IMPORTANT:** The IBM Operational Decision Manager (ODM) installed is a non-persistent IBM ODM deployment.
 
+**Note:** Make sure to add an image policy in ICP console (Manage->Resource Security->Image Policies) to allow pulling images from `docker.io/bitnami/redis:*` registry.
```

Comment:
I would move this to the beginning of the Redis section since it affects pulling the Redis image down rather than here in the ODM section. 

Moreover, I would create a hyper link with the "add an image policy in ICP" text with the Knowledge Center instructions for doing so (https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.0/manage_images/image_security.html)

Reply:
Thanks for pointing this out. Should be fixed now.

Reply:
Looks good to me. Thanks